lib/Cake/basics.php

```diff
@@ -46,8 +46,8 @@ function config() {
 		$count = count($args);
 		$included = 0;
 		foreach ($args as $arg) {
-			if (file_exists(APP . 'Config' . DS . $arg . '.php')) {
-				include_once APP . 'Config' . DS . $arg . '.php';
+			if (file_exists(CONFIG . DS . $arg . '.php')) {
```

Comment:
Those should all then be DS inclusive for CONFIG:

    CONFIG . $arg

Note the consistency then towards the APP and all other paths.